examples/connect.rs

```diff
@@ -236,7 +221,8 @@ mod udp {
 fn read_stdin(mut tx: mpsc::Sender<Vec<u8>>) {
     let mut stdin = io::stdin();
     loop {
-        let mut buf = vec![0; 1024];
+        // Use a smaller buffer so that each chunk can fit in a UDP packet.
+        let mut buf = vec![0; 512];
```

Comment:
I don't get this change, isn't a UDP packet 65kb?

Also 1kb is used above.

Reply:
See above, I'll probably change this to 1500 bytes.